recipe/meta.yaml

```
@@ -64,11 +64,11 @@ requirements:
     - jsonpatch >=1.32
     - truststore >=0.8.0           # [py>=310]
     - archspec
+    - conda-libmamba-solver >=23.9.3
```

Once https://github.com/conda/conda-libmamba-solver/issues/350 is released, we can update this

```suggestion
    - conda-libmamba-solver >=23.11.0
```